Makefile

```diff
@@ -58,8 +58,8 @@ endif
 
 TYPES_FILES    = $(shell find pkg/apis -name types.go)
 GO_VERSION    ?= 1.13
-GOFLAGS		   =-mod=vendor
 
+export GOFLAGS=-mod=vendor
```

Comment:
I think these should probably add on to whatever `${GOFLAGS}` is already defined and available in the environment.